SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -260,36 +273,55 @@ void USpatialSender::SendComponentUpdates(UObject* Object, const FClassInfo& Inf
 
 	UE_LOG(LogSpatialSender, Verbose, TEXT("Sending component update (object: %s, entity: %lld)"), *Object->GetName(), EntityId);
 
-	ComponentFactory UpdateFactory(Channel->GetInterestDirty(), NetDriver);
+	USpatialLatencyTracer* Tracer = nullptr;
+#if TRACE_LIB_ACTIVE
```

Comment:
Perhaps we should modify GetTracer() to just return nullptr if the lib isn't active. Thoughts?

Reply:
Yeah I thought about this yesterday, would be good to remove a few of these